tests/conftest.py

```diff
@@ -129,6 +130,22 @@ def secret(request) -> SecretType:
     return request.param
 
 
+@pytest.fixture(scope="session")
```

Comment:
Those fixtures may be moved to `test_authentication_strategy_jwt.py` since we only use them from there.

Reply:
I've eliminated them entirely since they would only refer to constants.